docs/ingestion/ingestion-spec.md

```diff
@@ -324,7 +324,7 @@ An example `granularitySpec` is:
 }
 ```
 
-A `granularitySpec` can have the following components:
+A `granularitySpec` uses the same format as [query granularity](../querying/granularities.md) can have the following components:
```

Comment:
```suggestion
A `granularitySpec` can have the following components:
```
wrong spot for this :D

Reply:
Oops! Sorry 😅

Reply:
@techdocsmith where should this line be?